DeltNU-iOS/App/Directory/View/DirectoryView.swift

```diff
@@ -55,9 +55,11 @@ struct DirectoryView: View {
                 .resignKeyboardOnDragGesture()
             }
             .sheet(isPresented: $showingMember) {
-                MemberView(member: self.viewModel.members.first { member in
+                if let brother = self.viewModel.members.first(where: { member in
```

Comment:
So my guess would be the if let isnt triggering, so sheet is being presented, but MemberView isnt showing. And we get empty sheet

Reply:
Next step could be to dig into the viewModel and find out why viewModel.members.first is returning null